.github/CODEOWNERS

```diff
@@ -0,0 +1,181 @@
+llvm/unittests/FileCheck/* @llvm/pr-subscribers-testing-tools
+llvm/unittests/ProfileData/* @llvm/pr-subscribers-pgo
+llvm/utils/FileCheck/** @llvm/pr-subscribers-testing-tools
+llvm/utils/count @llvm/pr-subscribers-testing-tools
+llvm/utils/lit/* @llvm/pr-subscribers-testing-tools
+llvm/utils/not @llvm/pr-subscribers-testing-tools
+llvm/utils/split-file @llvm/pr-subscribers-testing-tools
+mlir/** @llvm/pr-subscribers-mlir
+mlir//AMDGPU/ @llvm/pr-subscribers-mlir-amdgpu
+mlir//AMX/ @llvm/pr-subscribers-mlir-amx
+mlir//Affine/ @llvm/pr-subscribers-mlir-affine
+mlir//Arith/ @llvm/pr-subscribers-mlir-arith
+mlir//ArmNeon/ @llvm/pr-subscribers-mlir-neon
+mlir//ArmSME/ @llvm/pr-subscribers-mlir-sme
+mlir//ArmSVE/ @llvm/pr-subscribers-mlir-sve
+mlir//AsmParser/ @llvm/pr-subscribers-mlir-core
+mlir//Async/ @llvm/pr-subscribers-mlir-async
+mlir//Bufferization/ @llvm/pr-subscribers-mlir-bufferization
+mlir//Bytecode/ @llvm/pr-subscribers-mlir-core
+mlir//Complex/ @llvm/pr-subscribers-mlir-complex
+mlir//ControlFlow/ @llvm/pr-subscribers-mlir-cf
+mlir//DLTI/ @llvm/pr-subscribers-mlir-dlti
+mlir//Debug/ @llvm/pr-subscribers-mlir-core
+mlir//EmitC/ @llvm/pr-subscribers-mlir-emitc
+mlir//Func/ @llvm/pr-subscribers-mlir-func
+mlir//IR/ @llvm/pr-subscribers-mlir-core @llvm/pr-subscribers-mlir-core @llvm/pr-subscribers-mlir-core
+mlir//IRDL/ @llvm/pr-subscribers-mlir-irdl
+mlir//Index/ @llvm/pr-subscribers-mlir-index
+mlir//LLVM @llvm/pr-subscribers-mlir-llvm
+mlir//MLProgram/** @llvm/pr-subscribers-mlir-mlprogram
+mlir//Math/ @llvm/pr-subscribers-mlir-math
+mlir//MemRef/ @llvm/pr-subscribers-mlir-memref
+mlir//NVGPU/ @llvm/pr-subscribers-mlir-nvgpu
+mlir//OpenACC/ @llvm/pr-subscribers-mlir-openacc
+mlir//OpenMP/ @llvm/pr-subscribers-mlir-openmp
+mlir//PDL @llvm/pr-subscribers-mlir-pdl
+mlir//Parser/ @llvm/pr-subscribers-mlir-core
+mlir//Pass/ @llvm/pr-subscribers-mlir-core
+mlir//Quant/ @llvm/pr-subscribers-mlir-quant
+mlir//Reducer/ @llvm/pr-subscribers-mlir-core
+mlir//SCF/ @llvm/pr-subscribers-mlir-scf
+mlir//SPIRV/ @llvm/pr-subscribers-mlir-spirv
+mlir//Shape/ @llvm/pr-subscribers-mlir-shape
+mlir//SparseTensor/ @llvm/pr-subscribers-mlir-sparse
+mlir//Support/ @llvm/pr-subscribers-mlir-core
+mlir//Tensor/ @llvm/pr-subscribers-mlir-tensor
+mlir//Tosa/ @llvm/pr-subscribers-mlir-tosa
+mlir//Transforms/ @llvm/pr-subscribers-mlir-core
+mlir//UB/ @llvm/pr-subscribers-mlir-ub
+mlir//tools/ @llvm/pr-subscribers-mlir-core
+mlir/Bindings @llvm/pr-subscribers-mlir-bindings
+mlir/ExecutionEngine @llvm/pr-subscribers-mlir-execution-engine
+mlir/GPU @llvm/pr-subscribers-mlir-gpu @llvm/pr-subscribers-mlir-async
+mlir/TableGen @llvm/pr-subscribers-mlir-ods
+mlir/Vector @llvm/pr-subscribers-mlir-vector
+mlir/linalg @llvm/pr-subscribers-mlir-linalg
+mlir/tblgen @llvm/pr-subscribers-mlir-ods
+mlir/tools/** @llvm/pr-subscribers-mlir-core
+runtimes/** @llvm/pr-subscribers-libcxx @llvm/pr-subscribers-libcxxabi @llvm/pr-subscribers-libunwind
```

Comment:
Discourse formatted my post interpreting `*` for bolding...

I edited my message there, but here they are again

```
pr-subscribers-mlir mlir/**
pr-subscribers-mlir-core mlir/**/IR/** mlir/**/IR/** mlir/**/Support/**  mlir/**/Parser/**  mlir/**/IR/**  mlir/**/Bytecode/**  mlir/**/AsmParser/** mlir/**/Pass/** mlir/**/tools/**  mlir/**/Reducer/**  mlir/**/Transforms/**  mlir/**/Debug/** mlir/tools/**
pr-subscribers-mlir-ods mlir/**TableGen** mlir/**tblgen**
pr-subscribers-mlir-bindings mlir/**Bindings**
pr-subscribers-mlir-gpu mlir/**GPU**
pr-subscribers-mlir-amdgpu mlir/**/AMDGPU/**
pr-subscribers-mlir-amx mlir/**/AMX/**
pr-subscribers-mlir-affine mlir/**/Affine/**
pr-subscribers-mlir-arith mlir/**/Arith/**
pr-subscribers-mlir-neon mlir/**/ArmNeon/**
pr-subscribers-mlir-sme mlir/**/ArmSME/**
pr-subscribers-mlir-sve mlir/**/ArmSVE/**
pr-subscribers-mlir-async mlir/**/Async/**
pr-subscribers-mlir-bufferization mlir/**/Bufferization/**
pr-subscribers-mlir-complex mlir/**/Complex/**
pr-subscribers-mlir-cf mlir/**/ControlFlow/**
pr-subscribers-mlir-dlti mlir/**/DLTI/**
pr-subscribers-mlir-emitc mlir/**/EmitC/**
pr-subscribers-mlir-func mlir/**/Func/**
pr-subscribers-mlir-async mlir/**GPU**
pr-subscribers-mlir-irdl mlir/**/IRDL/**
pr-subscribers-mlir-index mlir/**/Index/**
pr-subscribers-mlir-llvm mlir/**/**LLVM**
pr-subscribers-mlir-linalg mlir/**linalg**
pr-subscribers-mlir-mlprogram mlir/**/MLProgram/**
pr-subscribers-mlir-math mlir/**/Math/**
pr-subscribers-mlir-memref mlir/**/MemRef/**
pr-subscribers-mlir-nvgpu mlir/**/NVGPU/**
pr-subscribers-mlir-openacc mlir/**/OpenACC/**
pr-subscribers-mlir-openmp mlir/**/OpenMP/**
pr-subscribers-mlir-pdl mlir/**/PDL**
pr-subscribers-mlir-quant mlir/**/Quant/**
pr-subscribers-mlir-scf mlir/**/SCF/**
pr-subscribers-mlir-spirv mlir/**/SPIRV/**
pr-subscribers-mlir-shape mlir/**/Shape/**
pr-subscribers-mlir-sparse mlir/**/SparseTensor/**
pr-subscribers-mlir-tensor mlir/**/Tensor/**
pr-subscribers-mlir-tosa mlir/**/Tosa/**
pr-subscribers-mlir-ub mlir/**/UB/**
pr-subscribers-mlir-vector mlir/**Vector**
pr-subscribers-mlir-execution-engine mlir/**ExecutionEngine**
```

Reply:
You may replace trailing `/**` with `/` and replace `**Vector**` with `*Vector*`

Reply:
I suspect I would want `mlir/**/*Vector` actually